flyway/postgre/migrations/sql/V1_3__Create_lesson2_tables.sql

```diff
@@ -0,0 +1,98 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+do $$
+begin
+	if not exists (select 1 from PG_TYPE where TYPNAME = 'organization_status_type') then
+		create type organization_status_type
+		as enum ('active', 'suspend', 'deleted');
+	end if;
+	if not exists (select 1 from PG_TYPE where TYPNAME = 'user_status_type') then
+		create type user_status_type
+		as enum ('active', 'deactivated', 'deleted');
+	end if;
+	if not exists (select 1 from PG_TYPE where TYPNAME = 'gender_type') then
+		create type gender_type
+		as enum ('male', 'female');
+	end if;
+end $$;
+
+CREATE TABLE IF NOT EXISTS sso_organization (
+    organization_id SERIAL NOT NULL,
+    organization_name VARCHAR(255) NOT NULL,
+    status organization_status_type DEFAULT 'active'::organization_status_type NOT NULL,
+    address VARCHAR(255),
+    contact_number VARCHAR(255),
+    email VARCHAR(255),
+    country VARCHAR(255),
+    state VARCHAR(255),
+    city VARCHAR(255),
+    postal_code VARCHAR(255),
+    website_url VARCHAR(255),
+    kvp JSONB,
+    date_created TIMESTAMPTZ DEFAULT now(),
+    date_modified TIMESTAMPTZ DEFAULT now(),
+    
+    constraint sso_organization_pk primary key (organization_id)
+);
+
+CREATE TABLE IF NOT EXISTS sso_role (
+    role_id uuid NOT NULL,
```

Comment:
Đặt thêm default để ID này có thể tự sinh nếu câu INSERT không gen sẵn uuid truyền vào.
```suggestion
    role_id uuid NOT NULL DEFAULT uuid_generate_v4(),
```